monai/visualize/img2tensorboard.py

```diff
@@ -206,7 +224,12 @@ def plot_2d_or_3d_image(
 
     if d.ndim >= 4:
         spatial = d.shape[-3:]
-        for j, d3 in enumerate(d.reshape([-1] + list(spatial))[:max_channels]):
+        d = d.reshape([-1] + list(spatial))
+        if d.shape[0] == 3 and max_channels == 3 and has_tensorboardx and isinstance(writer, SummaryWriterX):  # RGB
+            writer.add_video(tag, d[None], step, fps=max_frames, dataformats="NCHWT")
```

Comment:
not sure why `max_frames` is used as the fps parameter, is it a typo?

Reply:
Hi @wyli ,

I think the `max_frames` parameter of GIF is same as FPS of video if we plot the 3D image as video, not correct?

Thanks.

Reply:
I updated the PR for FPS in the latest commit.

Thanks.